.github/workflows/new_package.yml

```diff
@@ -25,7 +25,8 @@ jobs:
           # The Python script prints the package name so that we can easily
           # echo it to $GITHUB_ENV. Alternatively we could parse if with a
           # regular string from the JSON string
-          pkg_name=$(./scripts/utils/create_package_template_from_json.py '${{ steps.issue-parser.outputs.jsonString }}')
+          echo '${{ steps.issue-parser.outputs.jsonString }}' > pkg.json
```

Comment:
Sorry, I didn't review this PR very carefully 🙈 This won't fix https://github.com/mandiant/VM-Packages/issues/67 The problem is the character `'` in the description. Reopening https://github.com/mandiant/VM-Packages/issues/67 We need to remove this because the generated file will be added to the commit. 😖 

Still, the use of `ArgumentParser` in this PR is a nice addition 😄